src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -1,7 +1,7 @@
 ﻿---
 ################################
 # Assembly references report
-# Report date/time: 2023-04-24T16:00:30.3059365Z
+# Report date/time: 2023-05-02T09:14:46.3435124Z
```

Comment:
It doesn't do any harm to check these files in, but I don't normally bother if the only change is the date/time and file path (if anything else has changed then we have to check them in otherwise the CI build will fail).

I wouldn't bother removing the files from this PR, though.

Reply:
My bad, in the individual commits it looked like more had changed, didnt realize the second commit reverted the first commits change.